src/components/MultiHopTrade/components/TradeInput/components/CollapsibleQuoteList.tsx

```diff
@@ -8,18 +8,19 @@ export type CollapsibleQuoteListProps = {
   width: string | number
   height: string | number
   isLoading: boolean
-} & CardProps
+  cardProps: CardProps
+}
 
-export const CollapsibleQuoteList = ({
+export const CollapsibleQuoteList: React.FC<CollapsibleQuoteListProps> = ({
   isOpen,
   width,
   height,
   isLoading,
-  ml,
-}: CollapsibleQuoteListProps) => {
+  cardProps,
```

Comment:
Since we only actually used one prop from `cardProps` here, might as well keep `ml` explicit?

Reply:
gm acad9c0094b1483f6ff1bbf45b6e07f12504b5f6